proposals/0132-sequence-end-ops.md

```diff
@@ -3,8 +3,9 @@
 * Proposal: [SE-0132](0132-sequence-end-ops.md)
 * Authors: [Brent Royal-Gordon](https://github.com/brentdax), [Dave Abrahams](https://github.com/dabrahams)
 * Review Manager: [Chris Lattner](http://github.com/lattner)
-* Status: **Deferred**
-* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution-announce/2016-July/000267.html)
+* Status: **Awaiting Review** (Draft 2)
+* Implementation: [apple/swift#3793](https://github.com/apple/swift/pull/3793) 
+* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/3abbed3edd12dd21061181993df7952665d660dd/proposals/0132-sequence-end-ops.md)
```

Comment:
The use of “plagued” below strikes me as a tad over-dramatic.  There is a problem; there is room for improvement; overselling the impact of the problem hurts the argument.